### PR TITLE
Add a usage example

### PR DESCRIPTION
**Pull request by jrgns**
I'm not sure what the plans are for updating plugin docs, but a usage section like this can be helpful to those new to Logstash.

Thoughts, comments?

### CODE REVIEW

**Comment by suyograo** [comment]
@jrgns thanks, this is useful! can you sign the CLA? https://github.com/elasticsearch/logstash/blob/master/CONTRIBUTING.md#contribution-steps

File: lib/logstash/outputs/email.rb [hunk 1]
```diff
@@ -4,6 +4,24 @@
 
 # Send email when an output is received. Alternatively, you may include or
 # exclude the email output execution using conditionals. 
+#
+# ### Usage Example
```
Comment (suyograo) [inline lib/logstash/outputs/email.rb]: Please use `==== Usage Example` instead of `###`. We are using asciidoc instead of markdown

**Comment by jrgns** [comment]
@suyograo Thanx for the feedback. I've signed the CLA, but committed under the wrong email address. That should be resolved now.

**Comment by purbon** [comment]
@jrgns @suyograo CLA is ok, however we might need this PR to be rebased before getting this in. Could you also fix the small change @suyograo requested you to be asciidoc ?

Thanks a lot, looking forward to get this in.

**Comment by jrgns** [comment]
@purbon @suyograo I rebase the commit, and made the changes as requested :)

**Comment by talevy** [comment]
@jrgns looks like another rebase is needed. do you mind rebasing and we can go ahead and merge this in?

**Comment by jrgns** [comment]
@talevy Done. I've cleaned up the change a bit as well.

File: lib/logstash/outputs/email.rb [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+#       to => 'technical@logstash.net'
+#       from => 'monitor@logstash.net'
+#       subject => 'Alert - %{title}'
+#       body => "Tags: %{tags}\\n\\Content:\\n%{message}"
+#       domain => 'mail.logstash.net'
+#       port => 25
+#     }
+#   }
+# }
+# ~~~
```
Comment (suyograo) [inline lib/logstash/outputs/email.rb]: `-----------------------------------------`

File: lib/logstash/outputs/email.rb [hunk 3]
```diff
@@ -1,3 +1,19 @@
+#
+# ==== Usage Example
+# ~~~
```
Comment (suyograo) [inline lib/logstash/outputs/email.rb]: This should actually be 
```
[source,js]
----------------------------------
```

**Comment by suyograo** [comment]
@jrgns code blocks should follow: https://github.com/elastic/docs/blob/master/README.asciidoc#code-blocks

Can you make the change? I'll merge it in

**Comment by jrgns** [comment]
@suyograo I've updated it, no worries. Let me know if that's fine. Not sure why the CLA check isn't working?

**Pull request merged**